wpilibc/athena/include/DriverStation.h

```diff
@@ -68,6 +68,7 @@ class DriverStation : public SensorBase, public RobotStateInterface {
   Alliance GetAlliance() const;
   int GetLocation() const;
   void WaitForData();
+  bool WaitForData(long long timeout);
```

Comment:
That would be a breaking change on the Java side, as it has had this method in the public DriverStation API for a while. However, its not a feature I think many teams use, so we might be OK changing that. What do you think?